app/views/SecuritiesRequestedStatements.scala.html

```diff
@@ -32,66 +31,30 @@
 
 @layout(pageTitle = Some(messages("cf.security-statements.requested.title")), backLink = Some(backLink)) {
 
-@h1(id=Some("security-statements-heading"), classes="govuk-heading-xl govuk-!-margin-bottom-3", msg=messages("cf.security-statements.requested.title"))
+@h1(id=Some("security-statements-heading"),
+  classes="govuk-heading-xl govuk-!-margin-bottom-3",
+  msg=messages("cf.security-statements.requested.title")
+)
 
-@p(classes="govuk-body govuk-!-margin-bottom-8", id=Some("available-text"), content=Html(messages("cf.security-statements.requested.available.text")))
+@p(classes="govuk-body govuk-!-margin-bottom-8",
+  id=Some("available-text"),
+  content=Html(messages("cf.security-statements.requested.available.text"))
+)
 
-@for(historyIndex <- statements.indices) {
-    @if(statements(historyIndex).requestedStatements.nonEmpty) {
-        @if(historyIndex > 0) {
+<dl class="govuk-summary-list statement-list">
```

Comment:
With <dl> outside of the for , it would be present on the page even when there are no statement and it would create issue especially in accessibility. Either keep a flag in ViewModel and make use of that in a conditional statement (to proceed to iterate over the statements or not ) or find other way not to dispaly <dl> when there is no statements